charts/kubescape-operator/README.md

```diff
@@ -678,3 +678,58 @@ classDef plain fill:#ddd,stroke:#fff,stroke-width:1px,color:#000
 class k8sApi k8s
 class pr plain
 ```
+
+# Automatic Release Upgrade
+
+The Helm chart provides a capability to automatically fetch the latest chart version and upgrade the installed release.
+
+## How to Use
+
+To enable this capability, simply simply set the `capabilities.autoUpgrading` to `enable` and configure how often you would like to check for updates by adjusting the cron schedule:
+
+```yaml
+capabilities:
+  autoUpgrading: enable
+
+# [Omitted for brevity...]
+
+autoUpgrader:
+  schedule: "0 14 * * *"  # Check for updates every day at 14:00
+```
+
+Helm will then install the Kubescape Helm Release Upgrader CronJob and its supporting resources.
+They will keep your Kubescape release up to date.
+
+> [!WARNING]
+> Due to how Helm works, the Helm Release Upgrader requires highly elevated RBAC permissions and leaves orphaned resources even after you uninstall the Kubescape release.
+> This may or may not fit your use case and security requirements.
+> Please read how the Release Upgrader works and consider the risks before you use it.
```

Comment:
Whoever reviews this, please take a look at the docs I wrote just to make sure we align on the message with this capability.

Reply:
I definitely think the message should be reviewed by product...